browser/net/brave_network_delegate_browsertest.cc

```diff
@@ -51,12 +61,17 @@ class BraveNetworkDelegateBrowserTest : public InProcessBrowserTest {
     return HostContentSettingsMapFactory::GetForProfile(browser()->profile());
   }
 
+  void SetUpCommandLine(base::CommandLine* command_line) override {
+    // This is needed to load pages from "domain.com" without an interstitial.
+    command_line->AppendSwitch(switches::kIgnoreCertificateErrors);
+  }
+
   void DefaultBlockAllCookies() {
     brave_shields::SetCookieControlType(
         browser()->profile(), brave_shields::ControlType::BLOCK, GURL());
   }
 
-  void BlockThirdPartyCookies() {
+  void BlockAllThirdPartyCookies() {
```

Comment:
`BlockAllThirdPartyCookies` doesn't mean anything different to me than `BlockThirdPartyCookies`. `DefaultBlockThirdPartyCookies` maybe?